### DISCUSSION TITLE
Enable configuring router endpoint with XML file (to enable interface whitelisting)

### DISCUSSION THREAD

**Discussion by Vicidel** [eProsima/DDS-Router#361] [Ideas]
Hello, I would like to be able to configure a local participant (or other participants) with an XML, would that be possible? My application would be to have the participant with FastDDS interface whitelisting enabled in order to listen on some specific interface.

If it's possible in some other way I would also appreciate it.

Best, Victor

**Comment by jparisu** [comment]
Hi @Vicidel .

That is a cool feature that we already have in mind.
Unfortunately, for the moment it is not possible to do it, and we do not have an estimated date for it.

We thought for the future feature to have a specific participant tag `profile` that will configure that participant with such profile loaded from XML. Bu we would love to hear about the community ideas.

**Reply by Vicidel** [reply]
I would very much like this, or at least applying an [interface whitelisting](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/whitelist.html) would be enough in my case.
